product-mini/platforms/common/libc_wasi.c

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (C) 2023 Amazon.com Inc. or its affiliates. All rights reserved.
```

Comment:
@loganek
it seems that the majority of product-mini/platforms/common/libc_wasi.c is a copy-and-paste from other files.
i guess you should have preserved the original copyright notice.
what do you think?